Dockerfile.node20-x86_64

```diff
@@ -0,0 +1,24 @@
+FROM public.ecr.aws/lambda/nodejs:20-x86_64
+
+ENV PATH=/var/lang/bin:/usr/local/bin:/usr/bin/:/bin:/opt/bin
+
+# Configure linker to correctly point to libraries
+ENV LD_LIBRARY_PATH="/usr/lib:/usr/lib64"
+RUN dnf install -y xorg-x11-fonts-* libSM.x86_64 libXinerama-devel google-noto-sans-cjk-fonts binutils tar gzip xz \
+                    openssl nss-tools dbus-libs cups-libs && dnf clean all
+
+RUN cp /lib64/libssl.so.3 /lib64/libssl3.so
+
+RUN mkdir ~/libre && cd ~/libre && curl -s -L https://download.documentfoundation.org/libreoffice/stable/7.6.7/rpm/x86_64/LibreOffice_7.6.7_Linux_x86-64_rpm.tar.gz | tar xvz
+
+RUN cd ~/libre/LibreOffice_7.6.7.2_Linux_x86-64_rpm/RPMS/ && rpm -Uvh *.rpm && rm -fr ~/libre
```

Comment:
It seems we're still on version 7.6. Should it be updated to 7.7?

Reply:
<img width="501" alt="Screenshot 2024-05-14 at 11 52 18" src="https://github.com/shelfio/libreoffice-lambda-base-image/assets/34674660/a1e4695d-646e-41be-815b-8beebebdd668">
 
I would go with 7.6.7 :D